src/Makefile

```diff
@@ -82,35 +83,51 @@ kak : $(objects)
 	gzip -f $(@:.gz=.1)
 	mv -f $(@:.gz=.1.gz) $@
 
+check: test
 test:
```

Comment:
I dont like having these two targets just for the sake of having the GNU one as well. We could either keep test, or rename it to check. I'd favor test, which seems more explicit, but I dont feel very strongly about that.

In general, following the GNU conventions just to follow them does not seems very important, they are nice to follow to stay consistent with other software, but only when they make sense.